pkg/controller/nodes/dynamic/handler.go

```diff
@@ -129,6 +129,8 @@ func (d dynamicNodeTaskNodeHandler) handleDynamicSubNodes(ctx context.Context, n
 	return trns, newState, nil
 }
 
+// The State machine for a dynamic node is as follows
```

Comment:
Did you mean to complete this paragraph ?

Reply:
ohh ya, fragmented the brain